portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/Search.testcase

```diff
@@ -385,6 +385,47 @@ definition {
 			searchTerm = "");
 	}
 
+	@description = "This is a use case for LPS-179990"
+	@priority = 3
+	test CanRemoveSortFieldsAndDeactivateRelevanceInSortWidget {
+		task ("Open search page and add a new Sort Widget") {
+			Search.openSearchPage();
+
+			SearchPortlets.addWidgetsViaUI(searchPortletList = "Sort");
+
+			Refresh();
+		}
+
+		task ("Open Sort Widget configuration") {
+			Portlet.gotoPortletOptions(
+				portletName = "Sort",
+				portletOption = "Configuration");
+
+			SelectFrame(locator1 = "IFrame#CONFIGURATION");
+		}
+
+		task ("Delete all indexed field names and disable relevance") {
+			var count = 0;
+
+			while (${count} != 6) {
+				Click(locator1 = "Button#TRASH_ENABLED");
+
+				var count = ${count} + 1;
+			}
+
+			KeyPress(
+				checkboxName = "On",
+				locator1 = "Checkbox#ANY_CHECKBOX_CHECKED",
+				value1 = "\ENTER");
+		}
```

Comment:
Let's avoid using `KeyPress` unless the test is specifically testing keyboard actions. This function can be finicky because it requires focus on the modal/element that it's interacting with.